src/main/java/frc/robot/Robot.java

```diff
@@ -106,4 +132,38 @@ public void testPeriodic() {
     public void testExit() {
 
     }
+
+    private Command coralLevel2Place(){
+        return new SequentialCommandGroup(
+                Commands.runOnce(()-> new LowerCoralElevator(coralElevator)),
```

Comment:
```suggestion
                new LowerCoralElevator(coralElevator),
```

no, this is likely done in response to my comments on using the arm joint. But this was only needed for the arm joint since it is controlled very differently and not via running commands.

For all the other subsystems, to make them work you just use the commands. Doing `Commands.runOnce(()-> new LowerCoralElevator(coralElevator))` doesn't the command `LowerCoralElevator`....

Fix else where